src/components/tree/phyloTree/helpers.js

```diff
@@ -101,6 +101,73 @@ export const setYValuesRecursively = (node, yCounter) => {
  */
 export const setYValues = (nodes) => setYValuesRecursively(nodes[0], 0);
 
+// sorts all child nodes from a subtree into their subtrees
+const collectSubtrees = (startNode, subtree, trait, currentTraitValue, subtreeStack) => {
+  if (!startNode.children) return;
+
+  for (let i = startNode.children.length-1; i >= 0; i--) {
+    let currentNode = startNode.children[i];
+    let thisNodeTraitValue = getTraitFromNode(currentNode.n, trait);
+    // todo: nodes with no trait value? should they be grouped together?
```

Comment:
I believe they should be ignored (cc @evogytis). This isn't quite possible to do currently for tips in auspice, but for branches we can set a `hidden` property so they won't be rendered.

Reply:
Yes, I reckon for the sake of simplicity just ignore nodes without trait values.

Reply:
Because ignoring those nodes resulted in them all being clustered at the top of the graph, I am actually explicitly hiding them (as well as the intermediate nodes that aren't relevant to the subtrees). I think this makes things look much cleaner.